libs/ui/inputs/src/lib/star-toggle/star-toggle.component.html

```diff
@@ -5,12 +5,15 @@
     (click)="toggle($event)"
     [ngClass]="{
       enabled: toggled,
-      'disabled opacity-45': !toggled,
```

Comment:
this is also going to have impacts on other places right?

Reply:
Yes, do you see another way to make the star black in the nav bar when untoggled to be consistent with the other buttons?

Reply:
Yes it also impacts the display in the results list, which actually fits the new mockups. So I also changed the color from primary to secondary there. Will have to check if this is inline with the current work on the cards.